modules/tensor_mechanics/src/materials/TwoStressPlasticity.C

```diff
@@ -4,17 +4,23 @@
 /*          All contents are licensed under LGPL V2.1           */
 /*             See LICENSE for full restrictions                */
 /****************************************************************/
-#include "PQPlasticModel.h"
+#include "TwoStressPlasticity.h"
```

Comment:
I think i'd rename ComuteReturnMappingStress to ComputeAdmissibleConfiguration.  We're not just computing stress, but also inelastic strains and other things, with the requirement that the overall configuration must be admissible.  What do you think @bwspenc and @sapitts ?

Then updateStress could also be called computeAdmissible or something similar.